address_test.go

```diff
@@ -760,3 +760,197 @@ func TestInvalidCashAddreTestVectors(t *testing.T) {
 		}
 	}
 }
+
+func TestDecodeCashAddressSlpMainnet(t *testing.T) {
+	addrStr := "simpleledger:qrkjty23a5yl7vcvcnyh4dpnxxzuzs4lzqvesp65yq"
+	prefix, data, err := bchutil.DecodeCashAddress(addrStr)
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+	if prefix != "simpleledger" {
+		t.Fatal("decode failed")
+	}
+	if len(data) != 34 {
+		t.Fatal("data wrong length")
+	}
+}
+
+func TestDecodeCashAddressSlpMainnetUpperCase(t *testing.T) {
+	addrStr := "SIMPLELEDGER:QRKJTY23A5YL7VCVCNYH4DPNXXZUZS4LZQVESP65YQ"
```

Comment:
NIT: maybe one upercase test without prefix too. But this looks great. =)

Reply:
thanks, just pushed